index.bs

```diff
@@ -1243,7 +1279,13 @@ operation.
 
 <div dfn-for=request>
 
+A [=/request=] has a <dfn>processed flag</dfn> which is initially unset.
+This flag is set when the operation associated with the request has been executed.
+
+A [=/request=] is said to be <dfn>processed</dfn> when its [=request/processed flag=] is set.
+
 A [=/request=] has a <dfn>done flag</dfn> which is initially unset.
+This flag is set when the request of the operation associated with the request are available.
```

Comment:
This line doesn't make sense to me

Reply:
Oops, should say "This flag is set when the results of the operation...."